### PR TITLE
Scenes App: Fix scenes dependency

### PR DESCRIPTION
**Pull request by matyax**
I followed the demo-app instructions and stumbed upon a bunch of errors telling ` Error: Can't resolve '@grafana/scenes' in '/Users/matyax/Apps/scenes/packages/scenes-app/src/pages'`.

Changing the dependency version fixed them and I was able to see the demo app.

### CODE REVIEW

**Comment by torkelo** [comment]
@matyax strange, where are you getting these error? the dependency is correctly set to workspace (ie it's using the version in the your local repo)

**Comment by torkelo** [comment]
@matyax try yarn install from repo root, then `yarn dev:app` to build the demo app (need to build the lib first with `yarn dev:lib` )

File: packages/scenes-app/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "@emotion/css": "^11.1.3",
     "@grafana/data": "canary",
     "@grafana/runtime": "canary",
-    "@grafana/scenes": "workspace:*",
```
Comment (torkelo) [inline packages/scenes-app/package.json:65]: workspace is correct

**Comment by matyax** [comment]
@torkelo With a fresh clone of the repo. Followed the instructions and the `scenes-app` could not resolve the scenes dependency with workspace. Was  I missing anything?

**Comment by matyax** [comment]
The problem was my `GRAFANA_PATH` variable.

**Comment by torkelo** [comment]
@matyax no I think your right the demo.sh script is missing building the library

**Pull request closed without merging**